tincr/io/design/tincr_checkpoints.tcl

```diff
@@ -441,8 +441,8 @@ proc write_static_and_routethrough_luts { site_list channel } {
         
         if { [regexp {(O[5,6])=[0,1] ?} $config -> pin] } { ; # GND/VCC source
             lappend static_sources "[get_property NAME $bel]/$pin"
-        } elseif { [regexp {O[5,6]=\((A[1-6])\) ?} $config -> pin] } { ; # LUT routethrough
-            lappend routethrough_luts "$bel/$pin"
+        } elseif { [regexp {(O[5,6])=\((A[1-6])\) ?} $config -> outpin inpin] } { ; # LUT routethrough
+            lappend routethrough_luts "$bel/$inpin/$outpin"
```

Comment:
A cheap way to test for a route-through! Very awesome.

Could you add this logic to the `bels::is_route_through` method [here](https://github.com/byuccl/tincr/blob/master/tincr/cad/device/bels.tcl)?